source/_integrations/hifiberry.md

```diff
@@ -0,0 +1,39 @@
+---
+title: HifiBerry
+description: How to set up the HifiBerry media player platform
+logo: hifiberry.png
```

Comment:
Please remove this line, as it is no longer used.

Reply:
which line sorry? I can see first four on your comment

Reply:
ah logo, I see